fixes #209 new links for nav

emullaney opened the pull request.

Commit:
fixes #209 new links for nav

kmtracey commented:
When I clicked on "Deployment toolkit" in the new menu I noticed it moved closer to its sibling...this was pre-existing behavior it seems...is that intentional? It seems odd to me. Given the additional local pages in submenus here it's now more noticeable and I think we should fix it here (unless it really is intentional)? The selected nav items for some reason lose their padding...

Commit:
refs #209 adjusting R/L padding css for subnav links

daaray commented:
There is an _ever_ so slight downward motion (1px maybe?) when I load the page `/` (which has no sub nav) and then click on one of the elements that loads a subnav (`Learn`, for example).

kmtracey committed:
Fix the one pixel jump...

...by giving subnav a minimum height. (This also avoids the delayed
appearance on page load.) Also make the multi-word subnav items not
wrap on narrow screens.
